channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -272,6 +275,8 @@ export class Sender extends BaseTelemetryPlugin implements IChannelControls {
                     // getExtCfg only finds undefined values from core
                     let senderConfig = ctx.getExtCfg(identifier, defaultAppInsightsChannelConfig);
                     let curExtUrl = senderConfig.endpointUrl;
+                    _statsBeat = core.getStatsBeat();
```

Comment:
You have several instances of `core.getStatsBeat()` where the `getStatsBeat` portion is not going to be able to be minified.

suggestion:
Create a local helper function to return this value.

```ts
function _getStatsBeat() {
    return _self.core.getStatsBeat();
}
```

And then just call this function where you currently are, this means that all of those usages will get minified into a couple of characters, you no longer need the `_core`.

Downside: there is 1 additional function call `_getStatsBeat()` -> `_self.core.getStatsBeat()`, so there is a slight perf impact of this, but as it's not called in any tight loops and only when we are dealing with network requests I think this would be ok.

Reply:
It's always a trade-off between bundle size and performance (as this is also performing 2 lookups `_self.core` and then `core.getStateBeat` and then the function call.